tests/FileSystem/paket.references

```diff
@@ -1,6 +1,3 @@
 FSharp.Core
-Fable.Core
-Expecto
-Fable.Mocha
 Microsoft.NET.Test.Sdk
```

Comment:
Why not remove these two dependencies?

Reply:
Otherwise they don't appear in the test explorer in vs